app/js/app-menu.js

```diff
@@ -602,12 +604,17 @@ module.exports = function() {
 
       // unlock graph topolpgy in case it is locked
       chiseInstance.elementUtilities.unlockGraphTopology();
-
       // reset map name and description
       // default map name should be a string that contains the network id
-      currentGeneralProperties.mapName = appUtilities.getDefaultMapName(networkId);
-      currentGeneralProperties.mapDescription = appUtilities.defaultGeneralProperties.mapDescription;
-    
+      if(!currentGeneralProperties.mapPD2AFConverted) {// if the file don't have a specified map name from pd map to af map conversion
+        currentGeneralProperties.mapName = appUtilities.getDefaultMapName(networkId);
+        currentGeneralProperties.mapDescription = appUtilities.defaultGeneralProperties.mapDescription;
+      }
+      // We temporarly set inferNestingOnLoad to true to preserve the nest relation in the pd2af conversion   
```

Comment:
It seems that comment doesn't match with the action.

Reply:
Done

Reply:
I didn't understand what is done. Comment says "We temporarly set inferNestingOnLoad to true" but below code sets it to false.

Reply:
This section is unnecessary. I'll delete it on the next commit